src/gui/sdl_mapper.cpp

```diff
@@ -760,32 +757,11 @@ class CStickBindGroup : public CBindGroup {
 		} else return nullptr;
 	}
 
-	bool CheckEvent(SDL_Event * event) override {
```

Comment:
Did you confirm that the mapper GUI still works after removing these `CheckEvent()` functions.  I believe when we spoke on Discord, you had confirmed the one place the mapper was using SDL events over polling was inside the mapper GUI.

Reply:
Yes mapper GUI still works. These functions were not used for mapper GUI.